src/control/cmd/dmg/man_test.go

```diff
@@ -0,0 +1,57 @@
+//
+// (C) Copyright 2020 Intel Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+// The Government's rights to use, modify, reproduce, release, perform, display,
+// or disclose this software are subject to the terms of the Apache License as
+// provided in Contract No. 8F-30005.
+// Any reproduction of computer software, computer software documentation, or
+// portions thereof marked with this legend must also reproduce the markings.
+//
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"flag"
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+)
+
+var update = flag.Bool("update", false, "update dmg manpage")
+
+func TestManPage(t *testing.T) {
```

Comment:
Per discussion in the Go channel a while ago, we're trying to make test names a bit more descriptive in order to more easily understand them in context when picking through the output of a full test run. Suggest calling this `TestDmg_ManPageIsCurrent`, as it both identifies the context as 'Dmg' and also includes the intent of the test.

Reply:
Will update it to batch that convention.